pipeline/clean/opuscleaner/configs/remove_frequent_patterns.txt

```diff
@@ -0,0 +1,9 @@
+# Starting text in a lot of rows in OpenSubtitles, not everytime in both source and targets
```

Comment:
Question: What is this file exactly?

Reply:
This file is required for one of the filters and it is a copy-paste from opus-cleaner. https://github.com/hplt-project/OpusCleaner/blob/main/opuscleaner/filters/remove_frequent_patterns.txt